tests/integration/test_orders_cli.py

```diff
@@ -452,197 +473,49 @@ def test_cli_orders_request_basic_success(expected_ids, id_string, invoke):
-        '--name',
-        'test',
-        '--id',
-        '4500474_2133707_2021-05-20_2419',
-        '--bundle',
-        'analytic',
-        '--item-type',
-        'PSOrthoTile',
-        '--clip',
-        aoi_file
-    ])
-    assert not result.exception
-
-    order_request = {
-        "name":
-        "test",
-        "products": [{
-            "item_ids": ["4500474_2133707_2021-05-20_2419"],
-            "item_type": "PSOrthoTile",
-            "product_bundle": "analytic",
-        }],
-        "tools": [{
-            'clip': {
-                'aoi': geom_geojson
-            }
-        }]
-    }
-    assert order_request == json.loads(result.output)
-
-
-# TODO: add tests of "create --pretty" (gh-491).
-@pytest.mark.parametrize(
-    "id_string, expected_ids",
-    [('4500474_2133707_2021-05-20_2419', ['4500474_2133707_2021-05-20_2419']),
-     ('4500474_2133707_2021-05-20_2419,4500474_2133707_2021-05-20_2420',
-      ['4500474_2133707_2021-05-20_2419', '4500474_2133707_2021-05-20_2420'])])
-@respx.mock
-def test_cli_orders_create_basic_success(expected_ids,
-                                         id_string,
-                                         invoke,
-                                         order_description,
-                                         write_to_tmp_json_file):
-    mock_resp = httpx.Response(HTTPStatus.OK, json=order_description)
-    respx.post(TEST_ORDERS_URL).return_value = mock_resp
-
-    request_result = invoke([
         'request',
         '--name=test',
-        f'--id={id_string}',
+        '--id=',
         '--bundle=analytic',
-        '--item-type=PSOrthoTile'
+        '--item-type=PSOrthoTile',
     ])
-    request_file = write_to_tmp_json_file(json.loads(request_result.output),
-                                          'orders.json')
-
-    result = CliRunner().invoke(cli.main,
-                                ['orders', 'create', str(request_file)],
-                                catch_exceptions=True)
+    assert result.exit_code == 2
+    assert 'Entry cannot be an empty string.' in result.output
 
-    assert result.exit_code == 0
-    assert json.loads(
-        result.output)['_links']['results'][0]['delivery'] == 'success'
 
+@pytest.mark.parametrize("geom_fixture",
+                         [('geom_geojson'), ('feature_geojson'),
+                          ('featurecollection_geojson')])
+def test_cli_orders_request_clip_success(geom_fixture,
+                                         request,
+                                         invoke,
+                                         geom_geojson):
 
-@pytest.mark.parametrize(
-    "id_string, expected_ids",
-    [('4500474_2133707_2021-05-20_2419', ['4500474_2133707_2021-05-20_2419']),
-     ('4500474_2133707_2021-05-20_2419,4500474_2133707_2021-05-20_2420',
-      ['4500474_2133707_2021-05-20_2419', '4500474_2133707_2021-05-20_2420'])])
-@respx.mock
-def test_cli_orders_create_basic_stdin_success(expected_ids,
```

Comment:
@kevinlacaille I was able to remove some of these tests without a hit to coverage. This is because the removed tests tested both the `request` command (which was fully tested already) and the `create` command, but the `create` command simply takes an input and submits it to the server - it doesn't care what is in the input.